zubhub_frontend/zubhub/src/components/form/tagsInput/TagsInput.jsx

```diff
@@ -70,20 +112,20 @@ export default function TagsInput({
     </CustomButton>
   ));
 
-  selectedTags = selectedTags?.map((tag, index) => (
+  const selectedTagsRender = selectedTags?.map((tag, index) => (
     <CustomButton
       onClick={() => onSelectedTagClick(index)}
       className={classes.button}
       primaryButtonStyle
       key={index}
       endIcon={<ClearRounded />}
     >
-      {prefix && `${prefix}`}
+      {prefix && `${prefix} `}
       {tag}
     </CustomButton>
   ));
 
-  remoteData = remoteData?.map((tag, index) => (
+  const remoteDataRender = remoteData?.map((tag, index) => (
```

Comment:
same as above

Reply:
I'm a little failing to understand you on this, please elaborate